packages/opal-common/opal_common/fetcher/providers/http_fetch_provider.py

```diff
@@ -93,21 +98,29 @@ async def _fetch_(self):
         return result
```

Comment:
this is a comment for lines 88-92:
are you sure aiohttp and httpx have the same api for these http methods?

Reply:
they both have functions for every one of the http method functions and works basically the same, checked for get and post which both worked as expected